### PR TITLE
2080 - motorcycle_accidents_vs_all_accidents_widget.py - use new format for items

### LINKED ISSUE #2080: Widget motorcycle_accidents_vs_all_accidents - new data structure
- [ ] This widget contains a stacked bar graph - modify structure accordingly 
Discuss with Chen (FE team) to make sure you're aligned (since she needs to present percentage but I believe she will expecting absolute numbers)
- [ ] Re-register widget

FE issue - waiting for API - hasadna/anyway-newsflash-infographics#541

### PR DESCRIPTION
**Pull request by ziv17**
Closes #2080

### CODE REVIEW

**Review by BarVolunteering — COMMENTED** [review]
Looks Good!

**Review by BarVolunteering — APPROVED** [review]
Looks good!

File: anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py [hunk 1]
```diff
@@ -13,7 +14,14 @@
 from typing import Dict
 from flask_babel import _
 
-# TODO: register?
+
+location_other = "location other"
+location_road = "location road"
+location_all_label = "all roads"
```
[the remark this answers is not in the document]
Comment (ziv17) [reply anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py:20]: Yes, they are constants, that are local to the widget.

File: anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py [hunk 2]
```diff
@@ -46,8 +53,8 @@ def motorcycle_accidents_vs_all_accidents(
         ).label(location_label)
 
         vehicle_label = "vehicle"
-        vehicle_other = "אחר"
-        vehicle_motorcycle = "אופנוע"
+        vehicle_other = VehicleCategory.OTHER.get_english_display_name()
```
Comment (atalyaalon) [inline anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py:56]: This is a bit confusion but VehicleCategory.OTHER is referred as "CBS Other category", but here it's referred as all other vehicles. @ziv17 do you think we should add a different constant here?

Comment (ziv17) [reply anyway/widgets/suburban_widgets/motorcycle_accidents_vs_all_accidents_widget.py:56]: @atalyaalon , you are right. The label here should be `all other vehicles`, or כל האחרים
right?

**Pull request merged**